### PR TITLE
README: update sample output from the tool

### PR DESCRIPTION
**Pull request by 3flex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,22 +62,20 @@ It operates on the abstract syntax tree provided by the Kotlin compiler.
...
-      the baseline are printed in the console.
+      If a baseline xml file is passed in, only new code smells not in the 
+      baseline are printed in the console.
     --config, -c
       Path to the config file (path/to/config.yml).
     --config-resource, -cr
       Path to the config resource on detekt's classpath (path/to/config.yml).
     --create-baseline, -cb
-      Treats current analysis findings as a smell baseline for further detekt runs.
+      Treats current analysis findings as a smell baseline for further detekt
```
Comment (Mauin) [inline README.md]: Maybe we should change this to: "[...] for future detekt runs."

File: README.md [hunk 2]
```diff
@@ -62,22 +62,20 @@ It operates on the abstract syntax tree provided by the Kotlin compiler.
 The CLI uses jcommander for argument parsing. Following parameters are shown when `--help` is entered:
 
 ```
-The following option is required: --input, -i
```
Comment (Mauin) [inline README.md]: `--input` is still required.

Comment (3flex) [reply README.md]: That doesn't show when running with the `--help` option. I was just copy-pasting the output from the tool verbatim. I'll add that to the text above the usage output.

File: README.md (hunk 1 above)
Comment (3flex) [reply README.md]: Makes sense, I'll update it.

**Pull request merged**